test/functional/ui_ansible_roles_controller_test.rb

```diff
@@ -11,4 +11,11 @@ class UiAnsibleRolesControllerTest < ActionController::TestCase
     res = JSON.parse @response.body
     assert_equal res['total'], res['results'].size
   end
+
+  test 'should destroy role' do
+    delete :destroy,
+           :params => { :id => @role.id },
+           :session => set_session_user
+    assert_response :success
+  end
```

Comment:
Can we add a test to verify that a user without the `destroy_ansible_roles` permission cannot perform this action?

Reply:
I am not sure if this is necessary. The permissions should already be tested by a test in core and I didn't really find an existing test that tested this.

Reply:
@MariaAga what is your thought about this?

Reply:
Not my area, but I couldnt find a test in core for it in core